dotnet/src/Connectors/Connectors.AI.OpenAI/AzureSdk/ClientBase.cs

```diff
@@ -266,8 +262,8 @@ private static CompletionsOptions CreateCompletionsOptions(string text, Complete
             FrequencyPenalty = (float?)requestSettings.FrequencyPenalty,
             PresencePenalty = (float?)requestSettings.PresencePenalty,
             Echo = false,
-            ChoicesPerPrompt = 1,
-            GenerationSampleCount = 1,
+            ChoicesPerPrompt = requestSettings.ResultsPerPrompt,
+            GenerationSampleCount = requestSettings.ResultsPerPrompt,
```

Comment:
FYI: must be greater than ChoicesPerPrompt - https://learn.microsoft.com/en-us/dotnet/api/azure.ai.openai.completionsoptions.generationsamplecount?view=azure-dotnet-preview

Maybe the documentation is wrong, if that's the case I would raise a bug/reach out to Azure OpenAI team

Reply:
Yeah, the greaterthan seems to be wrong, should be GreaterThen Or Equals.